utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
+    init()
+except ImportError:  # fallback so that the imported classes always exist
+    class ColorFallback():
+        __getattr__ = lambda self, name: ''
+    Fore = Back = Style = ColorFallback()
+
+def color_diff(diff):
+    '''Colorize lines in the diff generator object'''
+
+    for line in diff:
+        if line.startswith('+'):
+            yield Fore.GREEN + line + Fore.RESET
```

Comment:
I'd suggest to replace with`''.join((Fore.GREEN , line , Fore.RESET))` or even declare formatter like `mark_added = lambda line: ''.join((Fore.GREEN , line , Fore.RESET))` for better readability